server/lib/src/server/migrations.rs

```diff
@@ -1300,7 +909,7 @@ impl QueryServerReadTransaction<'_> {
 
 #[cfg(test)]
 mod tests {
-    use super::{ProtoDomainUpgradeCheckItem, ProtoDomainUpgradeCheckStatus};
+    // use super::{ProtoDomainUpgradeCheckItem, ProtoDomainUpgradeCheckStatus};
```

Comment:
again, why not just delete?

Reply:
Laziness.

Reply:
Which really means I've left it there to uncomment once I put back something that uses the DomainUpgradeCheckItem in future.